### DISCUSSION TITLE
Open all in tabs not working with TabMix Plus on WaterFox G6.0.9

### DISCUSSION THREAD

**Discussion by thescrapyarduk** [onemen/TabMixPlus#276] [General]
I've been using TabMix Plus for a long long time and also before WaterFox support added with no issues but now if I have a folder of bookmarks in WaterFox G6.0.9 tabs do not open as they should, but will load if selected one-by-one. Not really ideal if you have a folder of 20 or more regular bookmarks you use and the "open all tabs" option isn't doing anything.

If I disable TabMix Plus from the extension setting and reload WaterFox "open all tabs" from within a folder works as it should. Enable TabMix Plus again while in the same browser session , and "open all tabs" doesn't do anything.

I've tried going through my TabMix Plus settings to see an option has changed, but my settings are what they have always been and the "open all tabs" option from within a bookmark folder used to work perfectly.

Any help appreciated as its slowing me down somewhat and don't want to have to disable TabMix Plus just to open all tabs from within a bookmark folder.

Thanks

**Comment by onemen** [comment]
Duplicate of #275

**Reply by thescrapyarduk** [reply]
Installed the very latest dev release earlier today but that didn't work. Just installed the updated test build linked from posting with same issues and checked on a few folders with open all bookmarks. All working as it should again

Thanks for your help and the work around for code changes in WaterFox
